crypto/crypto.go

```diff
@@ -53,31 +53,11 @@ func StringToSignatureAlgorithm(s string) SignatureAlgorithm {
 	}
 }
 
-// HashAlgorithm is an identifier for a hash algorithm.
-type HashAlgorithm = hash.HashingAlgorithm
-
-const (
-	UnknownHashAlgorithm HashAlgorithm = hash.UnknownHashingAlgorithm
-	SHA2_256                           = hash.SHA2_256
-	SHA2_384                           = hash.SHA2_384
-	SHA3_256                           = hash.SHA3_256
-	SHA3_384                           = hash.SHA3_384
-)
-
-// StringToHashAlgorithm converts a string to a HashAlgorithm.
-func StringToHashAlgorithm(s string) HashAlgorithm {
-	switch s {
-	case SHA2_256.String():
-		return SHA2_256
-	case SHA3_256.String():
-		return SHA3_256
-
-	default:
-		return UnknownHashAlgorithm
-	}
-}
-
-// CompatibleAlgorithms returns true if the signature and hash algorithms are compatible.
+// CompatibleAlgorithms returns true if the signature and hash algorithms is a valid pair for a signing key
```

Comment:
After moving the hashing related stuff to `hash.go`, this function only contains ECDSA signing functions. I would like to rename the file to `sign.go`. I will proceed to rename it later to not complicate the review. Let me know if there is a concern with the renaming.